### PR TITLE
Django 1.11 support

### PR DESCRIPTION
**Pull request by yakky**
Dropped old Django / python version in the process

Visible changes: use `get_fieldsets` instead of `declared_fieldsets`
Invisible changes: beware of `Widget.format_values` which changed semantic *significantly* in 1.11

### CODE REVIEW

**Comment by yakky** [comment]
@stefanfoulis @czpython would you give this a try in your environment? I'm not 100% comfortable with some changes here (even if tests and depending apps work without changes)

File: app_data/forms.py [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
 from django.forms.models import modelform_factory, _get_foreign_key, BaseInlineFormSet, BaseModelFormSet
 from django.utils.safestring import mark_safe
 from django.utils import six
+from django.utils.six import with_metaclass
+
+
+try:
+    from django.forms.utils import pretty_name
+except ImportError:
+    def pretty_name(name):
```
[the remark this answers is not in the document]
Comment (yakky) [reply app_data/forms.py]: Looks legit
Thanks

File: test_app_data/__init__.py [hunk 2]
```diff
@@ -1,26 +1,22 @@
...
 from os import environ
 import django
 
 environ['DJANGO_SETTINGS_MODULE'] = 'test_app_data.settings'
 
 test_runner = None
 old_config = None
-
-try:
+if LooseVersion(django.get_version()) >= LooseVersion('1.9'):
```
[the remark this answers is not in the document]
Comment (yakky) [reply test_app_data/__init__.py]: I honestly don't remember why I took that approach
You comments look legit

**Comment by yakky** [comment]
@filwaitman PR updated

**Comment by filwaitman** [comment]
@yakky looks good to me. =]

**Comment by yakky** [comment]
@filwaitman thanks!
@czpython as this changes a lot of things, I would release as 0.2 (I can prepare the repo but I can't release myself)

**Comment by czpython** [comment]
@yakky ok. I also don't have pypi access.

**Comment by yakky** [comment]
@czpython ` aldryn` user has it

**Comment by yakky** [comment]
I bumped the version on the master branch

**Comment by czpython** [comment]
[[ https://media.giphy.com/media/26ufdipQqU2lhNA4g/giphy.gif | height = 100px ]]

**Comment by czpython** [comment]
Thanks!

**Comment by czpython** [comment]
@yakky released

**Comment by yakky** [comment]
@czpython nice!
now https://github.com/aldryn/aldryn-apphooks-config/pull/54 it's the last bit to ensure addons dependencies are ready for 1.11

**Pull request merged**